src/styles/_reset.scss

```diff
@@ -0,0 +1,5 @@
+div,
+body {
+  padding: 0;
+  margin: 0;
+}
```

Comment:
div doesn't have any margins/paddings by default, so you don't need to reset them.